unified_planning/model/fnode.py

```diff
@@ -270,3 +271,71 @@ def is_le(self) -> bool:
     def is_lt(self) -> bool:
         """Test whether the node is the LT operator."""
         return self.node_type == OperatorKind.LT
+
+    #
+    # Infix operators
+    #
+
+    def __add__(self, right):
+        return get_env().expression_manager.Plus(self, right)
+
+    def __radd__(self, left):
+        return get_env().expression_manager.Plus(left, self)
+
+    def __sub__(self, right):
+        return get_env().expression_manager.Minus(self, right)
+
+    def __rsub__(self, left):
+        return get_env().expression_manager.Minus(left, self)
+
+    def __mul__(self, right):
+        return get_env().expression_manager.Times(self, right)
+
+    def __rmul__(self, left):
+        return get_env().expression_manager.Times(left, self)
+
+    def __truediv__(self, right):
+        return get_env().expression_manager.Div(self, right)
+
+    def __rtruediv__(self, left):
+        return get_env().expression_manager.Div(left, self)
+
+    def __floordiv__(self, right):
+        return get_env().expression_manager.Div(self, right)
+
+    def __rfloordiv__(self, left):
+        return get_env().expression_manager.Div(left, self)
+
+    def __gt__(self, right):
+        return get_env().expression_manager.GT(self, right)
+
+    def __ge__(self, right):
+        return get_env().expression_manager.GE(self, right)
+
+    def __lt__(self, right):
+        return get_env().expression_manager.LT(self, right)
+
+    def __le__(self, right):
+        return get_env().expression_manager.LE(self, right)
+
+    def __neg__(self):
+        return get_env().expression_manager.Minus(0, self)
```

Comment:
I think we need to add `__pos__`, and `__invert__` (see https://www.oreilly.com/library/view/python-in-a/0596001886/re22.html)